src/components/OverviewCard.jsx

```diff
@@ -154,9 +77,9 @@ const OverviewCard = ({
   const handleBlur = (event) => {
```

Comment:
Delete unused `handleBlur` functions in `OverviewCard` and `FolderContent`

Reply:
used in d697a01